test/spec/spectest.cpp

```diff
@@ -102,23 +95,62 @@ SpecTest::CommandID resolveCommand(std::string_view Name) {
 
 // Helper function to parse parameters from json to vector of value.
 std::pair<std::vector<WasmEdge::ValVariant>, std::vector<WasmEdge::ValType>>
-parseValueList(const rapidjson::Value &Args) {
+parseValueList(simdjson::dom::array &Args) {
```

Comment:
Why does the `const` get removed? Do you need to change the elements inside Args?

Reply:
Ohh, my bad. The const keyword was causing some issues with the Ondemad parser, so had to remove it.
But I have checked now, it works well with the DOM parser. Will add it back

Reply:
I have added the const arguments back here : [4020806](https://github.com/WasmEdge/WasmEdge/pull/2659/commits/4020806f88ab6fa990867cae568026c0782518df)